src/main/java/io/reactivex/Flowable.java

```diff
@@ -5056,29 +5056,420 @@ private static void verifyPositive(long value, String paramName) {
+     *            a default value to return if this {@code BlockingObservable} emits no items
+     * @return the last item emitted by the {@code BlockingObservable}, or the default value if it emits no
+     *         items
+     * @see <a href="http://reactivex.io/documentation/operators/last.html">ReactiveX documentation: Last</a>
+     */
+    public final T blockingLast(T defaultValue) {
+        BlockingLastSubscriber<T> s = new BlockingLastSubscriber<T>();
+        subscribe(s);
+        T v = s.blockingGet();
+        if (v != null) {
+            return v;
+        }
+        return v != null ? v : defaultValue;
+    }
+    
+    /**
+     * Returns an {@link Iterable} that returns the latest item emitted by this {@code BlockingObservable},
+     * waiting if necessary for one to become available.
+     * <p>
+     * If this {@code BlockingObservable} produces items faster than {@code Iterator.next} takes them,
+     * {@code onNext} events might be skipped, but {@code onError} or {@code onCompleted} events are not.
+     * <p>
+     * Note also that an {@code onNext} directly followed by {@code onCompleted} might hide the {@code onNext}
+     * event.
+     * <dl>
+     *  <dt><b>Backpressure:</b></dt>
+     *  <dd>The operator consumes the source {@code Flowable} in an unbounded manner
+     *  (i.e., no backpressure applied to it).</dd>
+     *  <dt><b>Scheduler:</b></dt>
+     *  <dd>{@code blockingLatest} does not operate by default on a particular {@link Scheduler}.</dd>
+     * </dl>
+     *
+     * @return an Iterable that always returns the latest item emitted by this {@code BlockingObservable}
+     * @see <a href="http://reactivex.io/documentation/operators/first.html">ReactiveX documentation: First</a>
+     */
+    public final Iterable<T> blockingLatest() {
+        return BlockingFlowableLatest.latest(this);
+    }
+    
+    /**
+     * Returns an {@link Iterable} that always returns the item most recently emitted by this
+     * {@code BlockingObservable}.
+     * <p>
+     * <img width="640" height="490" src="https://github.com/ReactiveX/RxJava/wiki/images/rx-operators/B.mostRecent.png" alt="">
+     * <dl>
+     *  <dt><b>Backpressure:</b></dt>
+     *  <dd>The operator consumes the source {@code Flowable} in an unbounded manner
+     *  (i.e., no backpressure applied to it).</dd>
+     *  <dt><b>Scheduler:</b></dt>
+     *  <dd>{@code blockingMostRecent} does not operate by default on a particular {@link Scheduler}.</dd>
+     * </dl>
+     *
+     * @param initialValue
+     *            the initial value that the {@link Iterable} sequence will yield if this
+     *            {@code BlockingObservable} has not yet emitted an item
+     * @return an {@link Iterable} that on each iteration returns the item that this {@code BlockingObservable}
+     *         has most recently emitted
+     * @see <a href="http://reactivex.io/documentation/operators/first.html">ReactiveX documentation: First</a>
```

Comment:
It's a copy of 1.x: https://github.com/ReactiveX/RxJava/blob/1.x/src/main/java/rx/observables/BlockingObservable.java#L293